OtripleS.Web.Api/Models/Fees/Fee.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using OtripleS.Web.Api.Models.Users;
+using System;
+
+namespace OtripleS.Web.Api.Models.Fees
+{
+    public class Fee : IAuditable
+    {
+        public Guid Id { get; set; }
+        public string Label { get; set; }
+        public decimal Amount { get; set; }
+        public Guid CreatedBy { get; set; }
+        public User CreatedByUser { get; set; }
```

Comment:
Separate every property and it's related entity with a new empty line - for instance:
```csharp
public decimal Amount {get; set;}

public Guid CreatedBy {get; set;}
public User CreatedByUser {get; set;}

public Guid UpdatedBy {get; set;}
public User UpdatedByUser {get; set;}

```